osbs/build/plugins_configuration.py

```diff
@@ -507,6 +507,14 @@ def render_tag_from_config(self):
 
         self.pt.set_plugin_arg(phase, plugin, 'tag_suffixes', tag_suffixes)
 
+    def render_pull_base_image(self):
+        """Configure pull_base_image"""
+        phase = 'prebuild_plugins'
+        plugin = 'pull_base_image'
+
+        self.pt.set_plugin_arg(phase, plugin, 'parent_images_digests',
```

Comment:
Maybe we only want to set this if there is a value to set?

Reply:
updated